src/workloads/query/multiplanner/CompoundIndexes.yml

```diff
@@ -0,0 +1,347 @@
+        - keys: {tenantId: 1, x14: 1}
+        - keys: {tenantId: 1, x15: 1}
+        - keys: {tenantId: 1, x16: 1}
+        - keys: {tenantId: 1, x17: 1}
+        - keys: {tenantId: 1, x18: 1}
+        - keys: {tenantId: 1, x19: 1}
+        - keys: {tenantId: 1, x20: 1}
+        - keys: {tenantId: 1, x21: 1}
+        - keys: {tenantId: 1, x22: 1}
+        - keys: {tenantId: 1, x23: 1}
+        - keys: {tenantId: 1, x24: 1}
+        - keys: {tenantId: 1, x25: 1}
+        - keys: {tenantId: 1, x26: 1}
+        - keys: {tenantId: 1, x27: 1}
+        - keys: {tenantId: 1, x28: 1}
+        - keys: {tenantId: 1, x29: 1}
+        - keys: {tenantId: 1, x30: 1}
+        - keys: {tenantId: 1, x31: 1}
+        - keys: {tenantId: 1, x32: 1}
+        - keys: {tenantId: 1, x33: 1}
+        - keys: {tenantId: 1, x34: 1}
+        - keys: {tenantId: 1, x35: 1}
+        - keys: {tenantId: 1, x36: 1}
+        - keys: {tenantId: 1, x37: 1}
+        - keys: {tenantId: 1, x38: 1}
+        - keys: {tenantId: 1, x39: 1}
+        - keys: {tenantId: 1, x40: 1}
+        - keys: {tenantId: 1, x41: 1}
+        - keys: {tenantId: 1, x42: 1}
+        - keys: {tenantId: 1, x43: 1}
+        - keys: {tenantId: 1, x44: 1}
+        - keys: {tenantId: 1, x45: 1}
+        - keys: {tenantId: 1, x46: 1}
+        - keys: {tenantId: 1, x47: 1}
+        - keys: {tenantId: 1, x48: 1}
+        - keys: {tenantId: 1, x49: 1}
+        - keys: {tenantId: 1, x50: 1}
+        - keys: {tenantId: 1, x51: 1}
+        - keys: {tenantId: 1, x52: 1}
+        - keys: {tenantId: 1, x53: 1}
+        - keys: {tenantId: 1, x54: 1}
+        - keys: {tenantId: 1, x55: 1}
+        - keys: {tenantId: 1, x56: 1}
+        - keys: {tenantId: 1, x57: 1}
+        - keys: {tenantId: 1, x58: 1}
+        - keys: {tenantId: 1, x59: 1}
+        - keys: {tenantId: 1, x60: 1}
+        - keys: {tenantId: 1, x61: 1}
+        - keys: {tenantId: 1, x62: 1}
+        - keys: {tenantId: 1, x63: 1}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+
+- Name: SimpleMultiplannerWith64Indexes
+  LoadConfig:
+    Path: "../../../phases/query/Multiplanner.yml"
+    Key: QueryTemplate
+    Parameters:
+      active: [3]
+      nopInPhasesUpTo: *maxPhase
+      repeat: *queryRepeats
+      database: *db
+      collection: *coll
+      query: &query {
+        Filter: {
+          tenantId: {$eq : 0},
+          x1: {$lt: *selectivity},
+          x2: {$lte: 1},
+          x3: {$lte: 1}
+        }
+      }
+
+- Name: Hide48Indexes
+  Type: RunCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [4]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+        Database: *db
+        Operations:
+        - {OperationName: RunCommand, OperationCommand: {collMod: *coll, index: {hidden: true, keyPattern: {tenantId: 1, x17: 1}}}}
```

Comment:
You can reference indexes by name instead of keyPattern in coll mod.

And I think it might be a good idea to have indexes named the same way in all our workloads of this type.

If we do this, we can move this huge actor into phases file and re-use.

Reply:
Done. Moved `Hide48Indexes` and `Hide14Indexes`.